conf.py

```diff
@@ -69,6 +70,9 @@ def setup(app):
       branch = rtd_version
    elif rtd_version == 'pre-release':
       branch = "pre-release"
+
+      # Skip the injection of the language switcher.
+      del html_js_files
    else:
       branch = "old-version"
```

Comment:
What about the `old-version` case, do we need to consider this?

Reply:
This is covered by the case above: switcher is enabled there, since this version is always created from the last latest. So, we do have some translations in there.